### DISCUSSION TITLE
Aborted (core dumped)

### DISCUSSION THREAD

**Discussion by p-zmud** [madMAx43v3r/chia-plotter#724] [Q&A]
Hi!

Every few plots I have error:

chia_plot: /home/piotr/chia-plotter/include/chia/bitfield_index.hpp:45: std::pair<long unsigned int, long unsigned int> bitfield_index::lookup(uint64_t, uint64_t) const: Assertion `bitfield_.get(pos) && bitfield_.get(pos + offset)' failed.
Aborted (core dumped)

I'm running Ubuntu 20, 5 3600, 32 gb ram.

What I can do with it?

**Comment by alt-savage** [comment]
Are you overclocking or running DDR4 XMP mode?  For instance, I've run into similar fails trying to run DDR4 3600 on a comet lake. Had to clock down to DDR4-3000.  No problems with DDR4 3600 on my Ryzen 5800x
